contrib/stack/topsStack/mergeBursts.py

```diff
@@ -12,6 +12,7 @@
 import datetime
 import logging
 import argparse
+import pathlib
```

Comment:
This is no longer needed.

Reply:
This is correct. It is not needed.  Sorry my ignorance, but I haven't done many PRs.  Is this accepted already or do I need to do a new PR?

Reply:
@falkamelung you can just push to the repo from which you issued this PR and this PR will get updated automatically.